src/test/java/PlaylistTests.java

```diff
@@ -9,8 +9,37 @@ public class PlaylistTests extends BaseTest {
     public void createPlaylist() throws InterruptedException {
         HomePage homePage = new LoginPage(driver).login();
         int initialNumberOfPlaylists = homePage.getNumberOfPlaylist();
-        homePage.createPlaylist("name");
+        homePage.createPlaylist("Super Songs");
         int updatedNumberOfPlaylists = homePage.getNumberOfPlaylist();
-        Assert.assertEquals(updatedNumberOfPlaylists, initialNumberOfPlaylists+1, "Playlist not created");
+        Assert.assertEquals(updatedNumberOfPlaylists, initialNumberOfPlaylists + 1, "Playlist not created");
     }
-}
\ No newline at end of file
+
+    @Test
+
```

Comment:
**Please delete the empty line!**  "An Empty Line is a "Code Smell"

The short answer is, They’re more an annoyance than a problem.They won’t actually break anything-which is why your code continues to run-but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce.